src/django/pfb_analysis/views.py

```diff
@@ -96,6 +96,7 @@ class NeighborhoodBoundsGeoJsonViewList(APIView):
 
     pagination_class = None
     filter_class = None
+    permission_classes = (AllowAny,)
```

Comment:
It's probably more correct to use existing `DjangoModelPermissionsOrAnonReadOnly` for these endpoints, so that if we did add write operations to these endpoints we wouldn't accidentally give the world access. But I don't see that happening, so if its not a direct drop and replacement I'm not so worried about it.